applications/serial_lte_modem/doc/MQTT_AT_commands.rst

```diff
@@ -0,0 +1,296 @@
+.. _SLM_AT_MQTT:
+
+MQTT AT Commands
+****************
+
+The following commands list contains the AT commands used to operate the MQTT client:
+
+* `MQTT Connect #XMQTTCON`_
+* `MQTT Subscribe #XMQTTSUB`_
+* `MQTT Publish #XMQTTPUB`_
+
+----
+
+MQTT Connect #XMQTTCON
+======================
+
+The ``#XMQTTCON`` command allows you to connect to and disconnect from the MQTT broker.
+
+Set command
+-----------
+
+The set command allows you to connect to and disconnect from the MQTT broker.
+
+Syntax
+~~~~~~
+
+::
+
+   AT#XMQTTCON=<op>[,<client_id>,<username>,<password>,<url>,<port>[,<sec_tag>]]
+
+
+* The ``<op>`` parameter is an integer and can accept one the following values:
+
+  * ``0`` - Connect to the MQTT broker.
+  * ``1`` - Disconnect from the MQTT broker.
+
+* The ``<client_id>`` parameter is a string and it indicates the MQTT Client ID.
+* The ``<username>`` parameter is a string and it indicates the MQTT Client username.
+* The ``<password>`` parameter is a string and it indicates the MQTT Client password.
+* The ``<url>`` parameter is a string and it indicates the MQTT broker hostname.
+* The ``<port>`` parameter is an integer and it indicates the MQTT broker port.
+* The ``<sec_tag>`` parameter is an integer and it indicates the credential of the security tag used for establishing a secure connection.
+
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XMQTTEVT: <evt_type>,<result>
+
+* The ``<evt_type>`` value is an integer.
+  When ``0``, it indicates the acknowledgment of the connection request.
+* The ``<result>`` value is an integer and it can assume the following values:
+
+  * ``0`` - Connection succeded.
+  * *Negative Value* - Error code. It indicates the reason for the failure.
+
+Examples
+~~~~~~~~
+
+::
+
+   AT#XMQTTCON=1,"MyMQTT-Client-ID","","","mqtt.eclipse.org",1883
```

Comment:
@lats1980 Is the op parameter here correct? Shouldn't that be 1 for disconnect and 0 for connect? In case please check also the op parameters of the following examples.